src/platform-apps/channels/channel-view.tsx

```diff
@@ -11,12 +11,17 @@ export class ChannelView extends React.Component<Properties> {
   renderMessages() {
     return (
       <div className='messages'>
-        {this.props.messages.map((message) => (
-          <Message
-            key={message.id}
-            {...message}
-          />
-        ))}
+        {this.props.messages.map((message, index) => {
+          const isFirstFromUser = index === 0 || message.sender.userId !== this.props.messages[index - 1].sender.userId;
+
+          return (
+            <Message
+              isFirstFromUser={isFirstFromUser}
```

Comment:
not totally sure it makes sense for the message to know whether it is the first message from a user. how the overall message list is displayed feels more like the responsibility of the parent than the message itself. instead of passing in a property here, can we just use a class?

something like this:
```
<Message
  className={classNames('messages__message', { isFirstFromUser: 'messages__message-first-in-group' })}
```
and then in the stylesheet:
```
.messages__message: {
  &-first-in-group: {
    .message__time: {
       visibility: hidden;

       &:hover {
         visibility: visible;
       }
    }
  }
}
```